app_dart/lib/src/request_handlers/github/webhook_subscription.dart

```diff
@@ -233,8 +234,10 @@ class GithubWebhookSubscription extends SubscriptionHandler {
       throw BadRequestException('Malformed merge_group request:\n$rawRequest');
     }
 
-    final eventAction = request['action'] as String;
-    final headSha = (request['merge_group'] as Map<String, Object?>)['head_sha'] as String;
+    final mergeGroupEvent = MergeGroupEvent.fromJson(request);
```

Comment:
fun:
  final MergeGroupEvent(:mergeGroup, :action, :headSha) = MergeGroupEvent.fromJson(request);

but you don't use eventAction?

Reply:
ignore the second part; eventAction is used in the zippie

Reply:
Done.